src/mono/mono/mini/interp/transform.c

```diff
@@ -3223,7 +3223,9 @@ get_bb (TransformData *td, unsigned char *ip)
 		bb->index = td->bb_count++;
 		td->offset_to_bb [offset] = bb;
 
-		td->basic_blocks = g_list_append_mempool (td->mempool, td->basic_blocks, bb);
+                /* Add the blocks in reverse order */
+                if (make_list)
```

Comment:
any reason for not using `td->gen_sdb_seq_points` directly here ?

Reply:
Not really. Wasn't sure if there's other reasons why we might need a list of blocks.